src/Laravel/Module.php

```diff
@@ -15,6 +15,12 @@ class Module extends BaseModule
      */
     public function getCachedServicesPath(): string
     {
+        // This checks if we are running on a Laravel Vapor managed instance
+        // and sets the path to a writable one (services path is not on a writable storage in Vapor).
+        if(!is_null(env('VAPOR_MAINTENANCE_MODE', null))) {
```

Comment:
Shouldn't this be retrieved from the config file? 
[From the documentation](https://laravel.com/docs/7.x/configuration#configuration-caching)

> If you execute the `config:cache` command during your deployment process, you should be sure that you are only calling the `env` function from within your configuration files. Once the configuration has been cached, the `.env` file will not be loaded and all calls to the `env` function will return `null`.

Since I don't use Vapor, I have no clue how's working `VAPOR_MAINTENANCE_MODE` But I have seen on [some packages that they were using `$_ENV` or `$_SERVER`](https://github.com/livewire/livewire/pull/1051) to get Vapor information.